src/Watchers/Warden.Watchers.ServerStatus/PortAvailabilityWatcher.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace Warden.Watchers.ServerStatus
+{
+    using System;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Sockets;
+    using System.Threading.Tasks;
+    using global::Warden.Core;
+
+    public class PortAvailabilityWatcher : IWatcher
+    {
+        /// <summary>
+        /// Default name for this watcher.
+        /// </summary>
+        public const string DefaultName = "Port availability Watcher";
```

Comment:
After renaming the whole project to _Port Watcher_ it should be also be its default name.